extensions/adobe/experience/journeyOrchestration/stepEvents/journeyOrchestrationDispatcherFieldsMixin.schema.json

```diff
@@ -0,0 +1,64 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/journeyOrchestration/serviceEvents/dispatcher",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Journey Orchestration service Events: Dispatcher Fields",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/experience/journeyOrchestration/serviceEvents"
+  ],
+  "description": "This event is in relation with journey and dispatcher service event fields",
+  "definitions": {
+    "journeyServiceDispatcher": {
+      "properties": {
+        "https://ns.adobe.com/experience/journeyOrchestration/serviceEvents/dispatcher": {
+          "title": "Journey Orchestration dispatcher service event fields.",
+          "type": "object",
+          "properties": {
+            "https://ns.adobe.com/experience/journeyOrchestration/serviceEvents/dispatcher/journeyEventID": {
+              "title": "Journey-Event ID",
+              "type": "string",
+              "description": "The Event ID to which this event was mapped inside the controller."
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/serviceEvents/dispatcher/journeyProjectionID": {
+              "title": "Journey Projection ID",
+              "type": "string",
+              "description": "The projection id of the segment being used in journey"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/serviceEvents/dispatcher/eventType": {
+              "title": "Event Type",
+              "type": "string",
+              "description": "The type of dispatcher service event like error, discard",
+              "enum": [
```

Comment:
If we describe the enums here, then later if we add more values to the schema, it will always require a change, so if we can skip explicitly defining the enums here, it might be helpful later on. We do not want to enforce strict checking here since we ourselves control what goes into the schema.

Reply:
Agreed